bin/release_docker

```diff
@@ -37,14 +35,6 @@ while test $# -gt 0; do
     docker_release_image myst:alpine mysteriumnetwork/myst:${VERSION_TAG}
     docker_release_image myst:alpine mysteriumnetwork/myst:${VERSION_TAG}-alpine
     docker_release_image myst:ubuntu mysteriumnetwork/myst:${VERSION_TAG}-ubuntu
-
-    docker_release_image myst:alpine mysteriumnetwork/mysterium-node:${VERSION_TAG}
-    docker_release_image myst:alpine mysteriumnetwork/mysterium-node:${VERSION_TAG}-alpine
-    docker_release_image myst:ubuntu mysteriumnetwork/mysterium-node:${VERSION_TAG}-ubuntu
```

Comment:
Are we stopping support for `mysterium-node` image?
Because internet is full of instructions how to setup it like here https://github.com/mysteriumnetwork/node/wiki/Links

Maybe we should push dummy image which says "You should be migrate to `mysteriumnetwork/myst service --agreed-terms-and-conditions`"

Reply:
We will still have the building of `mysterium-node` images in the `release/0.2` branch, so images will be active. 
Placing a dummy image that recommends using newer version should be done only after we finally deprecate `0.2.*` versions and stop doing any fixes there. 
I think it should be done a bit latter.

Reply:
now You clarified